css-view-transitions-1/Overview.bs

```diff
@@ -61,82 +86,346 @@ urlPrefix: https://html.spec.whatwg.org/multipage/rendering.html; type: dfn;
+
+	1. Rendering unpaused, revealing the transition pseudo-elements.
+
+	1. <code>viewTransition.{{ViewTransition/ready}}</code> fulfills.
+
+	1. Pseudo-elements animate until finished.
+
+	1. Transition pseudo-elements removed.
+
+	1. <code>viewTransition.{{ViewTransition/finished}}</code> fulfills.
+
+	<div id="phases-diagram">
+		<spec-scaler canvaswidth="1920" canvasheight="1080" style="aspect-ratio: 1920/1080">
+			<iframe class="spec-slides" src="diagrams/phases/"></iframe>
+		</spec-scaler>
+		<p class="spec-slide-controls">
+			<button disabled>Previous</button>
+			<button disabled>Next</button>
+		</p>
+		<script type="module">
+			const root = document.querySelector('#phases-diagram');
+			const [previous, next] = root.querySelectorAll('.spec-slide-controls button');
+			const iframe = root.querySelector('iframe');
+			next.disabled = false;
+
+			const updateButtons = (slide) => {
+				next.disabled = !slide.hasNext;
+				previous.disabled = !slide.hasPrevious;
+			};
+
+			next.addEventListener('click', async () => {
+				const slide = iframe.contentDocument.querySelector('spec-slide');
+				await slide.next();
+				updateButtons(slide);
+			});
+			previous.addEventListener('click', async () => {
+				const slide = iframe.contentDocument.querySelector('spec-slide');
+				await slide.previous();
+				updateButtons(slide);
+			});
+		</script>
+	</div>
+
+## Transitions as an enhancement ## {#transitions-as-enhancements}
+
+	A key part of this API design is that an animated transition is an enhancement to a document state change.
+
+	That means, a failure to create a transition,
+	which can happen due to misconfiguration or device constraints,
+	will not prevent the developer's {{UpdateCallback}} being called,
+	even if it's known in advance that the animated transition cannot happen.
+
+	For example, if the developer calls {{ViewTransition/skipTransition()}} at the start of the [[#lifecycle|lifecycle]],
+	the steps relating to the animated transition, such as creating the pseudo-elements, will not happen.
+	However, the {{UpdateCallback}} will still be called.
+	It's only the transition that's skipped, not the whole state change.
+
+	If the DOM change should also be skipped, then that should be handled by another feature.
+	<code>{{NavigateEvent|navigateEvent}}.{{NavigateEvent/signal}}</code> is an example of a feature developers could use to handle this.
+
+	Although the transition API allows DOM changes to be asynchronous via the {{UpdateCallback}},
+	the transition API is not responsible for queuing or otherwise scheduling the DOM changes beyond the scheduling needed for the transition itself.
+	Some asynchronous DOM changes can happen concurrently (e.g if they're happening within independent components),
+	whereas others need to queue, or abort an earlier change.
+	This is best left to a feature or framework that has a more holistic view of the application.
+
+## Examples ## {#examples}
```

Comment:
Added a sequence of examples similar to https://developer.chrome.com/docs/web-platform/view-transitions/, with videos.